config/dev.exs

```diff
@@ -1,6 +1,7 @@
 use Mix.Config
 
-config :mongoose_push, loglevel: :debug
+config :mongoose_push, :logging, level: :error
```

Comment:
why change debug to error?

Reply:
Because it's annoying to deal with sheer volume of debug logs, when developer just wants to enter application's shell and actually see commands you execute. As developer I don't need debug logs more often then I need then, so I prefer to have to enable debug logs explicitly, instead of constantly switching them off ;)

Reply:
Thank you!

Reply:
Yeah, that ^ :D

Reply:
Yeah, now it's explicit. I couldn't find it in the PR description somehow ;-)
It's good to describe such things so they don't look like mistakes.